natgeo/mortar
# [SYTO-971+SYTO-972] Design tweaks for Most Read/Latest component

https://jira.natgeo.com/browse/SYTO-971
https://jira.natgeo.com/browse/SYTO-972

To test:
Pull SYTO-971+SYTO-972
Run npm test to verify tests/linting pass
Run gulp serve
Navigate to http://localhost:3000/contentPackages/
Verify layout
